pkg/agent/openflow/packetin.go

```diff
@@ -50,16 +51,37 @@ func (c *client) StartPacketInHandler(stopCh <-chan struct{}) {
 	err := c.SubscribePacketIn(uint8(ofprAction), ch)
 	if err != nil {
 		klog.Errorf("Subscribe PacketIn failed %+v", err)
+		return
 	}
+	packetInQueue := workqueue.NewNamed("packetIn")
```

Comment:
I don't think the workqueue has a bounded size? Should we be cautious about too may packet ins being produced and the queue size growing?

Reply:
I think the size won't be large because only traceflow packet will use this, but I accept that we should be cautious about the size.
Added a validation here.
```
		if packetInQueue.Len() < packetInQueueSize {
			packetInQueue.Add(pktIn)
		} else {
			klog.Warningf("Max packetInQueue size exceeded.")
		}
```